templates/overrides/semantic/webbasic/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,74 @@
+@{
+    ViewData["Title"] = "Home Page";
+}
+
+
+<div class="ui blue inverted vertical masthead center aligned segment">
+
+    <div class="ui text container">
+
+        <h1 class="ui inverted header">
+            MVC + Semantic UI
+        </h1>
+        <h2>If you build it, they will come.</h2>
+        <a class="ui huge secondary button" href="//semantic-ui.com/">
```

Comment:
As previously: It will fail if hosted on HTTPS as semantic domain does not support HTTPS. Should be full link